src/submission/models.py

```diff
@@ -35,6 +35,7 @@
 from django.utils.functional import cached_property
 from django.utils.html import mark_safe
 import swapper
+from rest_framework.fields import ChoiceField
```

Comment:
This import is not used.

Reply:
- [X] Removed unused import
- [X] Checked PEP8 (I think my new lines were fine, but this will also be handled by a formatter soonish?)
- [X] I have added documentation to the docs files and the model's help_text fields explaining how the synchronisation does/doesn't work